src/components/vm/vmDetailsPage.jsx

```diff
@@ -87,15 +87,29 @@ export const VmDetailsPage = ({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
+    const vmNameHeadline = (
```

Comment:
No need to keep this seperate. Just create a const like this:

```
    const vmHeadline = (
    <Flex spaceItems={{ default: 'spaceItemsSm' }} component='h2'>
            {vm.title && <FlexItem>{vm.title}</FlexItem>}
            <FlexItem style={{ fontSize: "var(--pf-global--FontSize--xl)">
                {vm.name}
            </span>
    </Flex>
```

I did not test it, please adjust if needed.

Reply:
Great, this is how spaceItems is used. This code does not the same, but I now understand the idea and will change mine accordingly.